frontend/components.js

```diff
@@ -85,25 +316,74 @@ class UnlockClass extends Component {
             },
             'Try demo wallet'
           )
+        )
+      )
+
+    const LoadByHardwareWalletSection = () =>
+      h(
+        'div',
+        {class: 'auth-section'},
+        h(
+          'div',
+          undefined,
+          'Hardware wallets porvide the best security level ' + 'for your Cryptocurrency.'
         ),
         h(
           'div',
-          {class: 'centered-row'},
+          undefined,
+          'CardanoLite supports ',
+          h('a', {href: 'https://trezor.io/', target: 'blank'}, 'Trezor model T.')
+        ),
+        h(
+          'div',
+          {class: 'centered-row margin-top'},
           h(
             'button',
             {
-              class: 'trezor-button fade-in-up',
               onClick: () => loadWallet({cryptoProvider: 'trezor'}),
             },
             h(
               'div',
               undefined,
-              h('span', undefined, 'I have '),
+              h('span', undefined, 'use '),
               h('span', {class: 'trezor-text'}, 'TREZOR')
             )
           )
         )
       )
+
+    const LoadByFileSection = () =>
+      h(
+        'div',
+        {class: 'auth-section'},
+        h('div', {class: 'centered-row'}, 'CardanoLite supports key files generated by Daedalus.'),
```

Comment:
I would check with @refi93 what text we want here. Because cardanolite do not operate with same addresses as daedalus.

Reply:
I would drop this message. Daedalus currently supports importing/exporting wallet as JSON only in developer mode and the import of a JSON from Daedalus won't work as expected for most users that are not aware of the differences between cardanolite and daedalus.

I would put a similar message only when exporting the wallet and only when the JSON import/export JSON feature makes it to the official version of Daedalus.